capstone/client/src/components/StoresPage.js

```diff
@@ -70,6 +81,96 @@ class StorePage extends Component {
     });
   }
 
+  /* 
+   * Displays a dialog prompting the user to specify a name for the list that 
+   * is going to be saved. 
+   */
+  onShare = () => {
+    this.setState({
+      shareDialog: {
+        display: true,
+        shareButtonDisabled: true,
+        error: true,
+        errorText: "This is a required field."
+      },
+      errorMessage: null,
+    });
+  }
+
+  /*
+   * Checks to see if the "Email" field is empty. If the field is empty, an 
+   * error message is displayed and the save button is disabled. 
+   */
+  onTextFieldChange = (event) => {
+    if (event.target.value.trim() === '') {
```

Comment:
Check if the email address is valid using the regex that you had previously?

Reply:
+1